docusaurus/docs/e2e-test-a-plugin/locating-elements.md

```diff
@@ -12,23 +12,25 @@ keywords:
 sidebar_position: 30
 ---
 
-# Locating elements
+# Locating elements for E2E tests
```

Comment:
Should we not use `e2e` rather than `E2E`

Reply:
Why do you think it should be `e22`? The word `e2e` is not found in either the Google Developer Style Guide or the Microsoft Developer Style Guide, so I consulted the top few pages of a Google search and found E2E as the most common variant. I would go with E2E unless you know a good reason not to. After the PRs are published, I can do a global search/replace to enforce consistency.

Reply:
also this: https://en.wikipedia.org/wiki/E2E